packages/ui/components/index-page/AddButton.tsx

```diff
@@ -0,0 +1,15 @@
+import { PlusOutlined } from '@ant-design/icons';
```

Comment:
Can we kill semicolons? 😅🙏

Reply:
@brettstack I prefer using semicolons, for me, it is like using "period" or "comma" in a sentence :D Hence had enabled this setting in `.prettierc`, this would automatically format, the dev needn't add the semi colon :)